parse/parse_test.go

```diff
@@ -1613,6 +1789,152 @@ func Test_SQL(t *testing.T) {
 			INNER JOIN (SELECT id as uid FROM users WHERE id = 1) ON p.author_id = uid;`,
 			err: parse.ErrUnnamedJoin,
 		},
+		{
+			name: "compound select",
+			sql:  `SELECT * FROM users union SELECT * FROM users;`,
+			want: &parse.SQLStatement{
```

Comment:
do you consider add a `format() string` or similiar funcion, so in testing, you can just compare to a string? I can be really handy.    That's #616